src/main/java/gov/nih/nci/evs/api/fhir/R4/CodeSystemProviderR4.java

```diff
@@ -860,15 +833,23 @@ private List<CodeSystem> findPossibleCodeSystems(
   /**
    * Returns the concept map for the specified details.
    *
-   * @param details the details
    * @param id the id
    * @return the concept map
    * @throws Exception the exception
    */
   @Read
-  public CodeSystem getCodeSystem(final ServletRequestDetails details, @IdParam final IdType id)
-      throws Exception {
+  public CodeSystem getCodeSystem(@IdParam final IdType id) throws Exception {
```

Comment:
Just thinking that we can probably back off some of this logging since it's only for this method and was helpful for developing.  Same thing around like 925.  recommend just commenting them out so the log is less chatty but you keep this for any future debugging.